lib/manageiq/api/client/identity.rb

```diff
@@ -0,0 +1,29 @@
+module ManageIQ
+  module API
+    module Client
+      class Identity
+        attr_accessor :userid
```

Comment:
Prefer username over userid (unless the api actually returns it that way, in which make it match).  The reason is that id many times implies an integer.